elevate-entity-management/index.js

```diff
@@ -9,7 +9,7 @@ const getDependencies = () => {
 
 const getPackageMeta = () => {
 	return {
-		basePackageName: 'entity-management',
+		basePackageName: 'entity_management',
```

Comment:
@Prajwal17Tunerlabs keep the naming conventions  as per standards, check other packages naming convention

Reply:
@VISHNUDAS-tunerlabs  can u please tell, why it is changed ?